.github/workflows/ci.yaml

```diff
@@ -27,10 +27,23 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v4
         with:
-          go-version: "1.22"
+          go-version: '1.22'
+
+      - name: Install gotestsum
+        run: go install gotest.tools/gotestsum@latest
+
+      - run: mkdir junit-xml
```

Comment:
I think we should name all steps (or just make the step below be a two-command thing with `run: |` and this be the first command). Not all of our CIs name all steps, but in workflows that do we should probably keep doing it.

Reply:
Sure! Done. (I was thinking based on `sdk-python` that you liked not naming trivial shell steps but this seems good.)

Reply:
I would support changing that CI to start naming everything too (that things are not named is a product of history I think)